telco-core/configuration/reference-crs-kube-compare/required/performance/PerformanceProfile.yaml

```diff
@@ -0,0 +1,56 @@
+# required
+# count: 1
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: {{ .metadata.name }}
+  annotations:
+    # Some pods want the kernel stack to ignore IPv6 router Advertisement.
+    kubeletconfig.experimental: |
+      {"allowedUnsafeSysctls":["net.ipv6.conf.all.accept_ra"]}
+spec:
+  cpu:
+    # node0 CPUs: 0-17,36-53
+    # node1 CPUs: 18-34,54-71
+    # siblings: (0,36), (1,37)...
+    # we want to reserve the first Core of each NUMA socket
+    #
+    # no CPU left behind! all-cpus == isolated + reserved
+    isolated: {{ .spec.cpu.isolated }} # eg 1-17,19-35,37-53,55-71
+    reserved: {{ .spec.cpu.reserved }} # eg 0,18,36,54
+  # Guaranteed QoS pods will disable IRQ balancing for cores allocated to the pod.
+  # default value of globallyDisableIrqLoadBalancing is false
+  globallyDisableIrqLoadBalancing: false
+  hugepages:
+    defaultHugepagesSize: 1G
+    pages:
+        {{- range .spec.hugepages.pages }}
+        - size: {{ .size  }}
+          count: {{ .count }}
+          {{if ne (index . "node") nil }}
+          node: {{ .node }}
+          {{- end }}
+        {{- end }}
+{{- if .spec.machineConfigPoolSelector }}
+  machineConfigPoolSelector:
+{{ .spec.machineConfigPoolSelector | toYaml | indent 4 }}
+#    # For SNO: machineconfiguration.openshift.io/role: 'master'
+#    pools.operator.machineconfiguration.openshift.io/worker: ''
+{{- end }}
+{{- if .spec.nodeSelector }}
+  nodeSelector:
+{{ .spec.nodeSelector | toYaml | indent 4 }}
+#    # For SNO: node-role.kubernetes.io/master: ""
+#    node-role.kubernetes.io/worker: ""
+{{- end }}
+  workloadHints:
+    realTime: false
+    highPowerConsumption: false
+    perPodPowerManagement: true
+  realTimeKernel:
+    enabled: false
+  numa:
+    # All guaranteed QoS containers get resources from a single NUMA node
+    topologyPolicy: "single-numa-node"
+  net:
+    userLevelNetworking: false
```

Comment:
that setting is false by default, so you should be looking for: either absence, or for this being false:
https://github.com/openshift/cluster-node-tuning-operator/blob/c2bb89366efdb2d0fb1f4ab1811d933917bd1700/pkg/apis/performanceprofile/v1/performanceprofile_types.go#L158